index.html

```diff
@@ -903,11 +928,13 @@ <h3>Setting base direction for inline or substring text</h3>
 	</div>
 
  	<div class="req" id="bidi_inline_auto">
-	<p class="advisement">It must be possible to also set the direction for a span to auto. This means that the base direction will be determined by examining the content itself. A typical approach here would be to set the direction based on the first strong directional character outside of any markup.</p>
+	<p class="advisement">It must be possible to also set the direction for a span to <span class="kw" translate="no">auto</span>. This means that the base direction will be determined by examining the content itself. A typical approach here would be to set the direction based on the first strong directional character outside of any markup.</p>
```

Comment:
span probably needs to be .kw too

Reply:
@r12a not sure I understand. It's already class `kw`. I used `span` instead of `code` or `kbd` because it's not actually the keyword here--it's being used somewhat casually as a synonym for first-strong. I think probably I'll revert this line for now.

Reply:
You put .kw around `auto`, but not around the word `span` (just before).

Reply:
Ahh!!! Although I think the word span there doesn't mean `<span>` but just "a range of text"?